tip-wlan/charts/wlan-port-forwarding-gateway-service/templates/deployment.yaml

```diff
@@ -71,6 +71,10 @@ spec:
               value: {{ .Values.externallyVisible.host }}
             - name: PF_GATEWAY_RELEASE_EXT_PORT
               value: "{{ .Values.externallyVisible.port }}"
+            - name: EXT_PORT_RANGE_START
```

Comment:
In your comment you mention that the env variable is called LOCAL_PORT_RANGE_START. Here you are referring to EXT_PORT_RANGE_START. Which one is correct?